test/e2e/resources/notebook_instance_lifecycle_config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstanceLifecycleConfig
+metadata:
+  name: $NOTEBOOK_INSTANCE_LFC_NAME
+spec:
+  notebookInstanceLifecycleConfigName: $NOTEBOOK_INSTANCE_LFC_NAME
+  onStart:
+    - content: cGlwIGluc3RhbGwgUElM #pip install PIL
```

Comment:
did you add this to the requirements file

Reply:
The comment is the base64 decoded string, I'll update the comment to clarify it.